queries/markdown_inline/highlights.scm

```diff
@@ -16,10 +16,10 @@
 ] @string.escape
 
 ; Conceal codeblock and text style markers
-((code_span_delimiter) @markup.raw
+((code_span_delimiter) @markup.raw.delimiter
   (#set! conceal ""))
 
-((emphasis_delimiter) @markup.strong
+((emphasis_delimiter) @conceal
```

Comment:
If this is reverted dont we need to have something for it when not concealed

Reply:
No, it'll inherit from the containing `(emphasis)` etc. nodes. I simply should not have changed this.

Reply:
The issue here is that this node is shared between _all_ text formatting nodes (strong, italic, underlined, struck-through). So the choices are
1. no separate highlighting;
2. introduce `@markup.strong.delimiter` etc. (and restructure this pattern to fold it into the individual patterns, which might break due to directives with multiple nodes)
3. introduce a new higher-level `@markup.delimiter` capture.

I don't like 3., and 2. seems like too much work (and potential for issues) for this. Hence my choice for 1.